docs/UserGuide.md

```diff
@@ -95,7 +95,7 @@ Format: `help`
 
 #### Adding a student: `add`
 
-Adds a student to the tuition address book.
+Adds a student to the address book.
```

Comment:
standardise to just use `address book`

Reply:
why not use TAB though, there are some instances below that we use TAB and TAB is an easier way to differentiate between the old AB3 and our application

Reply:
address book is like short for of tuition address book I suppose. TAB is more referring to the application rather than the address book list itself.

Reply:
anyway I'll just replace with TAB first

Reply:
then wouldn't it be clearer and more distinct to simply use `tuition address book`, I get that it is like a word longer but leaving it as `address book` seems like we didn't take note of it. Perhaps when we add a Glossary section we should clarify that all instances of `address book` used is referring to `tuition address book`

Reply:
fix here https://github.com/AY2122S1-CS2103T-F13-3/tp/pull/97/files